src/query/sortResults.ts

```diff
@@ -0,0 +1,91 @@
+import debug from 'debug'
+import get from 'lodash.get'
+import { InternalEntity } from 'src/glossary'
+import { OrderBy, SortDirection } from './queryTypes'
+
+const log = debug('sortResults')
+
+type FlatSortCriteria = [string[], SortDirection]
+
+export function warnOnIneffectiveSortingKeys(obj: Record<string, any>): void {
```

Comment:
This function is only referenced in this file. No need to export it:
```suggestion
function warnOnIneffectiveSortingKeys(obj: Record<string, any>): void {
```